src/components/shared/S3FilePreview.tsx

```
@@ -300,14 +679,29 @@ const S3FilePreview = ({
+              if (process.env.NODE_ENV === 'development') {
+                console.log('[S3FilePreview] Video metadata loaded for:', src);
+              }
+            }}
+            onError={() => {
+              setIsLoading(false);
+              if (process.env.NODE_ENV === 'development') {
+                console.warn('[S3FilePreview] Video load error for:', src);
+              }
+            }}
+          >
+            Tu navegador no soporta el tag de video.
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)